src/components/ClusterView.tsx

```diff
@@ -20,42 +20,50 @@
 import * as React from "react";
 import { LibraryContainer } from "./LibraryContainer";
 import { LibraryItem } from "./LibraryItem";
-import { ItemData } from "../LibraryUtilities";
+import { ItemData, TooltipText } from "../LibraryUtilities";
+import { Tooltip } from "react-tooltip";
+require("react-tooltip/dist/react-tooltip.css");
 
 export interface ClusterViewProps {
-    libraryContainer: LibraryContainer,
-    icon: any,
-    clusterType: string,
-    showItemSummary: boolean,
-    childItems: ItemData[]
+  libraryContainer: LibraryContainer;
+  icon: any;
+  clusterType: string;
+  showItemSummary: boolean;
+  childItems: ItemData[];
 }
 
-export class ClusterView extends React.Component<ClusterViewProps, undefined> {
-
+export class ClusterView extends React.Component<ClusterViewProps> {
     render() {
-        return (
-            <div className={"ClusterViewContainer"}>
-                <div className={`ClusterLeftPane ${this.props.clusterType}`}>
-                    <img className={"ClusterIcon"} src={this.props.icon} />
-                </div>
-                <div className={"ClusterRightPane"}>
-                    {this.getNestedElements()}
-                </div>
-            </div>
-        );
-    }
-
-    getNestedElements() {
-
-        let index = 0;
-        return this.props.childItems.map((item: ItemData) => {
-            return (<LibraryItem
-                key={index++}
-                libraryContainer={this.props.libraryContainer}
-                showItemSummary={this.props.showItemSummary}
-                data={item}
-            />);
-        });
-    }
+      const tooltipContent = TooltipText[this.props.clusterType as keyof typeof TooltipText]
+    return (
+      <div className={"ClusterViewContainer"}>
+        <div className={`ClusterLeftPane ${this.props.clusterType}`}>
+          <a id={this.props.clusterType} className={"tooltipWrapper"}>
+            <img className={"ClusterIcon"} src={this.props.icon} />
+          </a>
+          <Tooltip
+            anchorSelect={`#${this.props.clusterType}`}
+            place="bottom"
```

Comment:
Is the tooltip still showing on top of the image like the screenshot or is snapshot from before?

Reply:
Yes it is. My bad